packages/hadron-app-registry/src/app-registry.ts

```diff
@@ -133,7 +159,7 @@ class AppRegistry {
    *
    * @returns {Component} The component.
    */
-  getComponent(name) {
+  getComponent(name: string): unknown {
```

Comment:
I think the return type of this can be `React.JSXElementConstructor<unknown> | undefined`

Reply:
Thank you! :)